pkg/installer/media.go

```diff
@@ -93,12 +94,13 @@ type Media struct {
 	Label     string
 	InputFile string
 
-	mType      MediaType
-	s          *sys.System
-	ctx        context.Context
-	unpackOpts []unpack.Opt
-	bl         bootloader.Bootloader
-	outputFile string
+	mType       MediaType
+	s           *sys.System
+	ctx         context.Context
+	unpackOpts  []unpack.Opt
+	bl          bootloader.Bootloader
+	outputFile  string
+	rawDiskSize imginstall.DiskSize
```

Comment:
nit: Wouldn't be simpler running the convert and check in advance and simply store the required MiB in here?